contrib/init/sysvinit-debian/docker

```diff
@@ -103,54 +91,50 @@ case "$1" in
 			ulimit -p unlimited
 		fi
 
-		log_begin_msg "Starting $DOCKER_DESC: $BASE"
+		log_begin_msg "Starting $DOCKER_DESC: $NAME"
 		start-stop-daemon --start --background \
 			--no-close \
-			--exec "$DOCKERD" \
+			--exec "$DAEMON" \
 			--pidfile "$DOCKER_SSD_PIDFILE" \
 			--make-pidfile \
 			-- \
-			-p "$DOCKER_PIDFILE" \
-			$DOCKER_OPTS \
-			>> "$DOCKER_LOGFILE" 2>&1
+				-p "$DOCKER_PIDFILE" \
+				$DOCKER_OPTS \
+					>> "$DOCKER_LOGFILE" 2>&1
 		log_end_msg $?
-		;;
+	;;
```

Comment:
Looks like the `shfmt` settings we use doesn't like some of these formatting changes; https://ci-next.docker.com/public/blue/rest/organizations/jenkins/pipelines/moby/branches/PR-42420/runs/1/nodes/251/log/?start=0

You can check the format it wants with;  `shfmt -w -bn -ci -sr`
<img width="846" alt="Screenshot 2021-05-25 at 21 45 43" src="https://user-images.githubusercontent.com/1804568/119559444-f81e0280-bda2-11eb-93e0-2d16500c61f8.png">

(The change of order of `-p "$DOCKER_PIDFILE"` it expects is a bit odd though; looks like it's getting confused by something)

Reply:
hmm, since patch is just for aligning formatting to Debian's version (no idea which one was first) - to make diffing easier - I can move that one to my packager queue and drop it from this submission.